### DISCUSSION TITLE
Pull request: Varcel authorisation

### DISCUSSION THREAD

**Discussion by DevrG03** [anuraghazra/github-readme-stats#1898] [Q&A] [answered]
when I am trying to create a pull request, there are three checks running out of which two are passed but one related to Varcel authorisation is failed, can anyone help about this problem?
<img width="910" alt="Screenshot 2022-07-28 at 7 32 26 PM" src="https://user-images.githubusercontent.com/103804567/181524791-48c51884-815a-4938-849a-ffb9f7db7f3b.png">

**Comment by rickstaa** [comment, marked as answer]
Hey @Devrajsinh03, thanks for your question. As explained in the comment by the (Vercel bot](https://github.com/anuraghazra/github-readme-stats/pull/1897#issuecomment-1198172836), the Vercel action fails because I or @anuraghazra need to authorize it first. However, the deployment is unnecessary since your PR is related to theme testing. Sorry for the confusion. If you were talking about the theme preview action not displaying a preview of your theme, I just created a bug report for this (see #1899).

**Reply by DevrG03** [reply]
Okay thank you @rickstaa
